src/schedule/zephyr_ll.c

```diff
@@ -512,8 +512,8 @@ int zephyr_ll_scheduler_init(struct ll_schedule_domain *domain)
 	struct zephyr_ll *sch;
 
 	if (domain->type != SOF_SCHEDULE_LL_TIMER) {
-		tr_err(&ll_tr, "zephyr_ll_scheduler_init(): unsupported domain %u",
-		       domain->type);
+		tr_warn(&ll_tr, "zephyr_ll_scheduler_init(): unsupported domain %u",
+			domain->type);
 		return -EINVAL;
 	}
```

Comment:
why SOF tries to register DMA domain if no topology is using it?

Btw, SOF on i.MX with Zephyr will still use DMA domain. As @paulstelian97 says we are working on adding support for it. 

We tried (and still @LaurentiuM1234 is looking at it) to use timer domain but the sound has glitches so we consider having DMA domain with zephyr also.

Reply:
@lyakh sure, all good with this commit.

Reply:
@dbaluta btw, any update on DMA based scheduling with Zpehyr ? Most of the LLL timer logic can be reused, but key difference is thread will have different priority and be scheduled via DMA IRQ rather than timer IRQ.

Reply:
@paulstelian97 is working on it and we will see some patches after he returns from vacation. ETA 2 weeks.